### PR TITLE
Make commands lazily loaded

### PR DESCRIPTION
**Pull request by kefzce**
This is just a fix of the previous PR https://github.com/doctrine/DoctrineMongoDBBundle/pull/641

More info by link https://github.com/doctrine/migrations/pull/839

### CODE REVIEW

**Review by malarzm — COMMENTED** [review]
Thank you for the PR and for patience while I was getting to it! Beside one minor styling change would you mind targeting the 4.2.x branch? The change is an improvement that we'll be happy to release as a bugfix, but it's time we sunset 3.x series.

**Comment by kefzce** [comment]
> Thank you for the PR and for patience while I was getting to it! Beside one minor styling change would you mind targeting the 4.2.x branch? The change is an improvement that we'll be happy to release as a bugfix, but it's time we sunset 3.x series.

That makes sense to me, sure.
Later today I will rename my  branch to 4.2.x and rebase PR from 4.2.x target

**Comment by kefzce** [comment]
@malarzm should I also fix CI by duplicate 
```
    /**
     * @var string|null
     */
```
notation of inherited \Symfony\Component\Console\Command\Command?
If so, your Travis also tell me that you prefer single line notation instead, or it's just fine?)

**Comment by malarzm** [comment]
@kefzce one-line notation is the way to go, but also if you add it in a normal way `phpcbf` should be able to fix it for you (it's in `vendor/bin/` folder). In general we need to have green light from CI to merge a PR :)

File: Command/TailCursorDoctrineODMCommand.php [hunk 1]
```diff
@@ -24,12 +24,13 @@
  */
 class TailCursorDoctrineODMCommand extends Command implements ContainerAwareInterface
 {
+    /** @var string */
+    protected static $defaultName = 'doctrine:mongodb:tail-cursor';
     use ContainerAwareTrait;
```
Comment (malarzm) [inline Command/TailCursorDoctrineODMCommand.php:29]:
```suggestion
    use ContainerAwareTrait;
    
    /** @var string */
    protected static $defaultName = 'doctrine:mongodb:tail-cursor';
```

I am not sure why this was not caught by our coding standards, could you please change it as suggested? I hope the multi-line suggestion works :)

Comment (kefzce) [reply Command/TailCursorDoctrineODMCommand.php]: fixed

**Review by greg0ire — APPROVED** [review]

**Review by malarzm — APPROVED** [review]

**Comment by malarzm** [comment]
Awesome, thank you @kefzce!

**Pull request merged**